.github/workflows/nix-rebuild.yml

```diff
@@ -110,20 +83,17 @@ jobs:
           name: nix-qchem
       # Checkout merge commit as if PR would already be merged
       - uses: actions/checkout@v3
-        with:
-          ref: ${{ github.sha }}
-          repository: ${{ github.event.pull_request.base.repo.full_name }}
```

Comment:
There is a good chance that this won't work for cross-fork PRs. The `repository` line was explicitly required to fix this.

Reply:
I opened a cross fork PR here: https://github.com/Nix-QChem/NixOS-QChem/pull/272
It looks like the checkout action is doing what it is supposed to do:
Check under "Checking out the ref". It says : "/usr/bin/git checkout --progress --force refs/remotes/pull/272/merge".